test/unit/protocol/message_layer/StreamMessageV28.test.js

```diff
@@ -71,7 +71,7 @@ describe('StreamMessageV28', () => {
         })
         it('correctly serializes messages to v31', () => {
             const arr = [31, ['TsvTbqshTsuLg_HyUjxigA', 0, 1529549961116, 0, '', ''],
-                [null, null], StreamMessage.CONTENT_TYPES.MESSAGE, StreamMessage.ENCRYPTION_TYPES.NONE, '{"valid": "json"}', 0, null]
+                null, StreamMessage.CONTENT_TYPES.MESSAGE, StreamMessage.ENCRYPTION_TYPES.NONE, '{"valid": "json"}', 0, null]
```

Comment:
why it was [null, null] and now its just null? what if I'll put ['blabla', null]?

Reply:
First question: dunno. Second question: example will not pass validation.